### PR TITLE
gh-103266: Fix a typo in example code for bisect() function

### PR DESCRIPTION
**Pull request by arhadthedev**
https://github.com/search?q=repo%3Apython%2Fcpython%20Speilberg&type=code shows that the reported two occurrences are the only ones.

* Issue: gh-103266

### CODE REVIEW

**Comment by arhadthedev** [comment]
Do we backport typo fixes?

**Comment by PedanticHacker** [comment]
I didn't realize this typo/misspelling was already reported and a PR already made with a proper fix.

@arhadthedev, I hope this gets backported. It won't hurt to fix this for older Python docs besides 3.11, right?

**Comment by hugovk** [comment]
> Do we backport typo fixes?

Yes, to avoid conflicts.

File: Doc/library/bisect.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ records in a table::
     >>> Movie = namedtuple('Movie', ('name', 'released', 'director'))
 
     >>> movies = [
-    ...     Movie('Jaws', 1975, 'Speilberg'),
+    ...     Movie('Jaws', 1975, 'Spielberg'),
     ...     Movie('Titanic', 1997, 'Cameron'),
     ...     Movie('The Birds', 1963, 'Hitchcock'),
     ...     Movie('Aliens', 1986, 'Scott')
```
Comment (hugovk) [inline Doc/library/bisect.rst]: Ridley Scott directed the original [_Alien_](https://en.wikipedia.org/wiki/Alien_(film)) (1979) but James Cameron directed the sequel [_Aliens_](https://en.wikipedia.org/wiki/Aliens_(film)) (1986).
```suggestion
    ...     Movie('Aliens', 1986, 'Cameron')
```

File: Doc/library/bisect.rst [hunk 2]
```diff
@@ -228,7 +228,7 @@ records in a table::
     >>> pprint(movies)
     [Movie(name='The Birds', released=1963, director='Hitchcock'),
      Movie(name='Love Story', released=1970, director='Hiller'),
-     Movie(name='Jaws', released=1975, director='Speilberg'),
+     Movie(name='Jaws', released=1975, director='Spielberg'),
      Movie(name='Aliens', released=1986, director='Scott'),
```
Comment (hugovk) [inline Doc/library/bisect.rst]:
```suggestion
     Movie(name='Aliens', released=1986, director='Cameron'),
```

**Review by BlindChickens — APPROVED** [review]

**Review by hugovk — APPROVED** [review]

**Review by rhettinger — APPROVED** [review]

**Review by hauntsaninja — APPROVED** [review]
I guess it was Alien 1979 that was directed by Scott

**Pull request merged**